### PR TITLE
Combined FSTree versioning and log

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: pkg/local_object_storage/blobstor/fstree/fstree.go [hunk 1]
```diff
@@ -60,14 +61,28 @@ type writer interface {
...
+	// directories, enough for a single FSTree.
+	MaxDepth = 8
 
 	// combinedPrefix is the prefix that Protobuf message can't start with,
 	// it reads as "field number 15 of type 7", but there is no type 7 in
 	// the system (and we usually don't have 15 fields). ZSTD magic is also
 	// different.
 	combinedPrefix = 0x7f
+
+	// combinedLenSize is sizeof(uint32), length is a serialized 32-bit BE integer.
```
Comment (carpawell) [inline pkg/local_object_storage/blobstor/fstree/fstree.go:74]:
```suggestion
	// combinedLenSize is sizeof(uint32), length of a serialized 32-bit BE integer.
```
no?

**Review by carpawell — APPROVED** [review]
Conflicts.

File: pkg/local_object_storage/blobstor/fstree/fstree.go (hunk 1 above)
Comment (roman-khimov) [reply pkg/local_object_storage/blobstor/fstree/fstree.go:74]: No.

Comment (roman-khimov) [reply pkg/local_object_storage/blobstor/fstree/fstree.go:74]: OK, to be fixed.

**Pull request merged**